### PR TITLE
New README

### PR DESCRIPTION
**Pull request by jamilbk**
Fixes firezone/gtm#125

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,52 +28,115 @@ the branch tracking the latest 0.7 release.
...
+## Frequently asked questions (FAQ)
+
+> **Can I self-host Firezone?**
+
+Our [license](#license) won't stop you from self-hosting the entire Firezone
+product top to bottom, but we can't dedicated the resources to make this a
+smooth experience and therefore don't support self-hosting the control plane at
+this time.
+
+If you have a business case requiring an onprem installation of Firezone please
```
Comment (bmanifold) [inline README.md]: Minor grammatical nit...I think `onprem` either needs to have a space or a hyphen.

File: README.md [hunk 2]
```diff
@@ -30,52 +28,115 @@ the branch tracking the latest 0.7 release.
...
+
+- [elixir/](./elixir): Control plane API and admin UI
+- [rust/](./rust): Data plane components and Linux / Windows clients
+- [swift/](./swift/apple): macOS / iOS clients
+- [kotlin/](./kotlin/android): Android / ChromeOS clients
+- [website/](./website): Marketing website and product documentation
+
+## Frequently asked questions (FAQ)
+
+> **Can I self-host Firezone?**
```
Comment (bmanifold) [inline README.md]: What do you think of using `###` headers instead of quote blocks for these FAQs?  As I was viewing the rendered page in the browser it got a little tough for me to quickly glance and see what the questions were, because it looks like some of the answers include a quote block, but in reality those were new questions.

**Review by bmanifold — COMMENTED** [review]
Overall, looks good.  Just a couple of very small questions.

File: README.md (hunk 1 above)
Comment (jamilbk) [reply README.md]: 👍 Updated

File: README.md (hunk 2 above)
Comment (jamilbk) [reply README.md]: I had a similar thought, updated!

**Review by jefferenced — APPROVED** [review]
Great start!

**Review by bmanifold — APPROVED** [review]
LGTM!

**Review by jefferenced — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -30,52 +31,122 @@ the branch tracking the latest 0.7 release.
...
 
-## Features
+[Firezone](https://www.firezone.dev/?utm_source=readme) is an open source
+platform to securely manage remote access for any-sized organization. Unlike
+most VPNs, Firezone takes a granular, least-privileged approach to access
+management with group-based policies that control access to individual
+applications, entire subnets, and everything in between.
 
+<p align="center">
+  <img width="1439" alt="architecture" src="https://github.com/firezone/firezone/assets/167144/48cd6a1e-2f3f-4ca7-969a-fc5b33e13d1c">
```
Comment (AndrewDryga) [inline README.md:56]: I would remove this image, bad graphics are worse than no graphics at all, and this one feels a bit hard to follow.

File: README.md [hunk 4]
```diff
@@ -30,52 +31,122 @@ the branch tracking the latest 0.7 release.
...
+## Features
 
-### Anti-features
+Firezone is:
+
+- **Fast:** Built on WireGuard® to be
+  [3-4 times](https://wireguard.com/performance/) faster than OpenVPN.
+- **Scalable:** Deploy two or more gateways for automatic load balancing and
+  failover.
+- **Private:** Tunnels are peer-to-peer so packets never route through Firezone
```
Comment (AndrewDryga) [inline README.md]: @jamilbk we have cloud-hosted relays so not sure about this one, we can't see decapsulated packets but we can relay them for symmetric cone NAT's.

File: README.md [hunk 5]
```diff
@@ -30,52 +31,122 @@ the branch tracking the latest 0.7 release.
...
+## Contents of this repository
+
+This is a monorepo containing the full Firezone product, marketing website, and
+product documentation, organized as follows:
+
+- [elixir/](./elixir): Control plane API and admin UI
+- [rust/](./rust): Data plane components and Linux / Windows clients
+- [swift/](./swift/apple): macOS / iOS clients
+- [kotlin/](./kotlin/android): Android / ChromeOS clients
+- [website/](./website): Marketing website and product documentation
```
Comment (AndrewDryga) [inline README.md]: Maybe let's go a little bit more into details here so that people doing due diligence can better grasp the scale of the product:

```suggestion
- [elixir](./elixir): Control plane and internal Elixir libraries:
  - [elixir/apps/web](./elixir/apps/web): Admin UI;
  - [elixir/apps/api](./elixir/apps/api): API for Clients, Relays and Gateways.

- [rust/](./rust): Data plane and internal Rust libraries:
  - [rust/gateway](./rust/gateway): Gateway - VPN server based on WireGuard and deployed to your infrastructure;
  - [rust/relay](./rust/relay): Relay - STUN/TURN server;
  - [rust/linux-client](./rust/linux-client): Linux client.

- [swift/](./swift/apple): macOS / iOS clients.
- [kotlin/](./kotlin/android): Android / ChromeOS clients.
- [website/](./website): Marketing website and product documentation.

- [terraform/](./terraform): Terraform files for our cloud infrastructure:
- [terraform/modules/gateway-google-cloud-compute](./terraform/modules/gateway-google-cloud-compute): Terraform module that deploys Gateway to a Google Compute Regional Instance Group.
```

**Review by AndrewDryga — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (jamilbk) [reply README.md]: Updated!

**Review by jefferenced — APPROVED** [review]

File: README.md (hunk 5 above)
Comment (jamilbk) [reply README.md]: 👍

**Comment by jamilbk** [comment]
Refs #2519
